plugin/pxr/cmake/macros/testWrapper.py

```diff
@@ -233,11 +233,11 @@ def _runCommand(raw_command, stdout_redir, stderr_redir, env,
     # Add any envvars specified with --env-var options into the environment
     env = os.environ.copy()
     for varStr in args.envVars:
-        if varStr == 'PATH':
-            sys.stderr.write("Error: use --pre-path or --post-path to edit PATH.")
-            sys.exit(1)
         try:
             k, v = varStr.split('=', 1)
+            if k == 'PATH': 
+                sys.stderr.write("Error: use --pre-path or --post-path to edit PATH.")
+                sys.exit(1)
```

Comment:
Thanks for fixing this. I can follow up with the core USD team to see whether we should address this for USD tests in the Pixar repo as well.